frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/EditFieldDrawer/edit-fieldtype/EditDate/EditDate.tsx

```diff
@@ -118,6 +179,11 @@ export const EditDate = ({ field }: EditDateProps): JSX.Element => {
     },
   })
 
+  const watchAddParticularDayRestriction = useWatch({
+    control: control,
```

Comment:
nit: 
```suggestion
    control,
```

nit: any reason why we chose `useWatch` hook over `useForm#watch`?

Reply:
In the documentation, they mentioned that using `useWatch` hook is slightly more performant than use `watch`. This was their reasoning:
```Behaves similarly to the watch API, however, this will isolate re-rendering at the custom hook level and potentially result in better performance for your application.```
Link for reference: https://react-hook-form.com/api/usewatch